commit a9fa35e73bbb84ed754e575ba7f6b7d738cb1edb
Author: bartus <szczepaniak.bartek+github@gmail.com>
Date:   Thu Jun 13 18:07:30 2019 +0200

    Revert "[core] Add git branch name in version name"
    
    This reverts commit 6fc1f09aed23f230da2b889efe015773fd57eaee.

diff --git a/meshroom/__init__.py b/meshroom/__init__.py
index d91e3c1..8ee42b3 100644
--- a/meshroom/__init__.py
+++ b/meshroom/__init__.py
@@ -1,17 +1,6 @@
 __version__ = "2019.1.0"
 
 import os
-scriptPath = os.path.dirname(os.path.abspath( __file__ ))
-headFilepath = os.path.join(scriptPath, "../.git/HEAD")
-# If we are in a release, the git history will not exist.
-# If we are in development, we declare the name of the git development branch in the version name
-if os.path.exists(headFilepath):
-    with open (headFilepath, "r") as headFile:
-        data = headFile.readlines()
-        branchName = data[0].split('/')[-1]
-        # Add git branch name to the Meshroom version name
-        __version__ = __version__ + "-" + branchName
-
 # Allow override from env variable
 __version__ = os.environ.get("REZ_MESHROOM_VERSION", __version__)
 
